web/src/i18n/translations/en.json

```diff
@@ -42,5 +42,13 @@
         "programs": "Programs",
         "episodes": "Episodes",
         "cancel": "Cancel"
+    },
+    "settings": {
+        "categories": {
+            "audioLanguage": "Audio Language",
+            "subtitleLanguage": "Subtitle Language"
+            
+        },
+        "profileSetting": "Profile Setting"
```

Comment:
Use "Settings" and not the singular form of the word, and reevaluate the naming.

Don't think we need to refer to "Profile" visually for the user.